docs/en/ingest-management/elastic-agent/advanced-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[advanced-kubernetes-managed-by-fleet]]
+= Advanced {agent} configuration managed by {fleet}
+
+For basic {agent} managed by {fleet} scenarios follow the steps in <<running-on-kubernetes-managed-by-fleet>>.
+
+On managed {agent} installations it can be useful to provide the ability to configure more advanced options, such as the configuration of providers during startup. Refer to <<providers>> for more details.
+
+Following steps demonstrate above scenario:
+
+[discrete]
+== Step 1: Download the {agent} manifest
+
+include::run-container-common/download-elastic-agent.asciidoc[]
+
+[discrete]
+== Step 2: Create a new configmap
+
+[source,yaml]
+.Create a new configmap
+------------------------------------------------
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-node-datastreams
+  namespace: kube-system
+  labels:
+    k8s-app: elastic-agent
+data:
+  agent.yml: |-
+    providers.kubernetes_leaderelection.enabled: false
+    fleet.enabled: true
+    fleet.access_token: "<FLEET_ENROLLMENT_TOKEN>"
+---
+------------------------------------------------
+
+Notes:: 
+1. The above example demonstrates how to disable the `kubernetes_leaderelection` provider. For alternative scenarios you can follow the same procedure. For example, the following settings configure the `kubernetes` provider metadata enrichment:
+
+  [source,yaml]
+  .Example of configmap for kubernetes provider configuration
```

Comment:
This part does not seem to appear correctly:
![image](https://github.com/elastic/ingest-docs/assets/113898685/407a5c0b-99ea-4d2d-aca3-0dfc25b8897d)

Reply:
Maybe because of the extra space in the beginning of the sentence.

Reply:
Oops! I didn't notice that. Removing the two extra spaces at the beginning of each line in the code example should fix things.

Reply:
I had done this deliberately because it was a second examled and wanted to continue numbering of notes. NW can be reverted